src/main/java/com/suse/saltstack/netapi/datatypes/Token.java

```diff
@@ -31,4 +34,8 @@ public Date getStart() {
     public Date getExpire() {
         return expire;
     }
+
+    public List<String> getPerms() {
```

Comment:
@gborp: indeed I think you are right on both accounts. Naming-wise I would follow the "spirit" of the [API endpoint convention we talked about in PR 15](https://github.com/SUSE/saltstack-netapi-client-java/pull/15#issuecomment-74485966) - keep the original name unless there's a strong reason not to.

About missing Javadoc, we currently don't have it on any method on any class in the `datatype` package. The reason is that attributes are supposed to directly map to Salt's JSON (see above), thus any documentation is pure copy-and-paste exercise and it's not really needed. Even if at some point we decide to have it for completeness sake, I would suggest to add it to all classes in a separate PR, not on a case-by-case basis.

@isbm @renner: I propose to simply manually merge commit a8833da.